### DISCUSSION TITLE
Does useFieldArray support falsy default values?

### DISCUSSION THREAD

**Discussion by jhackett1** [react-hook-form/react-hook-form#12026] [General]
I’m trying to use `useFieldArray` to render a set of email inputs.

It works great, but I’d like the initial state to be three empty inputs.

When I set the default values of the form to be `emails: ["", "", ""]` then I don’t see any inputs on the screen logging the `fields` variable returned by the hook proves it to be empty.

I assume this is because the values are falsy, since if `emails: ["a", "b", "c"]` then I _do_ see inputs appear.

How can I get what I want?

[Possibly related to this](https://github.com/orgs/react-hook-form/discussions/12018)

**Comment by tatsmaki** [comment]
It should contain objects, not primitives.
```
emails: [{ value: '' }, { value: '' }, { value: '' }]
```
